### DISCUSSION TITLE
Valve type survey

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#64] [Polls]
What kind of valve are you using ?
And are you open with diy valves ?

**Comment by grericht** [comment]
cheapest way i found: https://de.elv.com/p/dnt-5er-set-heizkoerperthermostat-thermotune-stand-alone-ca-4-jahre-batterielaufzeit-P253148/?itemId=253148
5pcs = 50eur -> 10eur per valve
1pc = 13eur

very easy to open and desolder and still usable without "case". the only bad thing: internal cable is only 10cm - u might have to solder a longer cable on it
